web/docs/Backend Starters/Decoupled WordPress/managing-dependencies.md

```diff
@@ -0,0 +1,30 @@
+# Managing WordPress Dependencies
+
+## 1. Adding a new plugin/theme:
+
+This WordPress Project uses [composer](https://getcomposer.org) to manage dependencies. A new plugin/theme can be added using the following composer require command:
```

Comment:
I think we need a generic example, followed by an actual example. So first:

`composer require <namespace>/<packagename>`

and then the akismet example.